tablet-report-parser/moses.pl

```diff
@@ -97,15 +103,20 @@ sub Get_and_Parse_tablets_from_tservers{
       }
       $n->{Tserver_UUID} ||= "*Unknown\@Idx-" . $n->{nodeIdx} . "*"; # Can happen for un-initialized system
       my $tabletCount = 0;
-      print "SELECT '", TimeDelta("Processing tablets on $n->{nodeName} $n->{Tserver_UUID} (Idx $n->{nodeIdx})..."),"';\n";
+      print "SELECT '", TimeDelta("Processing tablets on $n->{nodeName} $n->{Tserver_UUID} ($n->{private_ip},Idx $n->{nodeIdx})... $prev_node_msg"),"';\n";
       my $html_raw = $YBA_API->Get("/proxy/$n->{private_ip}:$n->{tserverHttpPort}/tablets?raw","BASE_URL_UNIVERSE",1); # RAW
-
+      if ( $opt{GZIP} ){
```

Comment:
It's not intuitive that a flag called GZIP produces raw output instead of sqlite. Maybe rename this flag?

Reply:
Documented `--gzip` usage in --help, and in the README.